romeo/asset-contract/tests/clarity-bitcoin-mini-deploy.test.ts

```diff
@@ -0,0 +1,21 @@
+import { beforeEach, describe, expect, it } from "vitest";
+
+beforeEach(async () => {
+  await vm.initSession(process.cwd(), "./Clarinet.toml");
+});
+
+describe("test deploy version and unit test version of clarity-bitcoin-mini", () => {
+  it("ensures the versions differ only in DEBUG flag", () => {
```

Comment:
Appreciate the comment here - answers my question on what's the difference b/w both contract.